kafka-ui-api/src/main/java/com/provectus/kafka/ui/config/ClustersProperties.java

```diff
@@ -19,7 +19,7 @@
     String name;
     String bootstrapServers;
     String zookeeper;
-    String schemaRegistry;
+    SchemaRegistry schemaRegistry;
```

Comment:
could we support both object and string scenarios?

Reply:
Resolved